src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -267,6 +267,10 @@ public function update_item( $request ) {
 
 		$request->set_param( 'context', 'edit' );
 
+		$post = get_post( $template->wp_id );
+		/** This action is documented in wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php */
+		do_action( "rest_after_insert_{$this->post_type}", $post, $request, false );
```

Comment:
I find it confusing that `rest_after_insert_` is fired also after `wp_update_post`, but it's in line with how the filter is documented so 🤷